index.js

```diff
@@ -959,7 +963,6 @@ function modifyNewtab() {
             // if we couldn't check if we can do the auto import because we weren't able to run the canUndo function, throw an error, and don't modify the newtab page with anything
             } catch(e) {
                 console.error('Not able to resolve autoimport undo promise.' + e);
-                worker.port.emit('modify', headerContent, footerContent);
```

Comment:
Removing, as this was there for debugging and I forgot to remove it in the last PR.